script.js

```diff
@@ -0,0 +1,10 @@
+function sum() {
+    var a = document.getElementById('a').value
+    var b = document.getElementById('b').value
```

Comment:
Minor naming note, we should always name things what they are. Something like `number1` or `input1` instead of `a` can be very helpful. Matters less with this small amount of code but as your code base becomes hundred or files and thousands of variables, keeping track becomes harder and harder without good consistent naming.